src/chainparams.h

```diff
@@ -115,9 +115,9 @@ class CChainParams
     bool fAllowMultiplePorts;
     CCheckpointData checkpointData;
     ChainTxData chainTxData;
-    int nPoolMinParticipants{0};
-    int nPoolMaxParticipants{0};
-    int nFulfilledRequestExpireTime{0};
+    int nPoolMinParticipants;
+    int nPoolMaxParticipants;
```

Comment:
nPoolMaxParticipants is not set for regtest params.

Reply:
https://github.com/dashpay/dash/pull/3007/commits/9e2235a4bc86b5e3de89655c5d2f948630e404c4